xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/DXVAHD.cpp

```diff
@@ -324,16 +318,39 @@ bool CProcessorHD::InitProcessor()
         DXGI_COLOR_SPACE_RGB_FULL_G22_NONE_P709, &supported);
     const bool topLeft = SUCCEEDED(hr) && static_cast<bool>(supported);
 
+    // Check if driver prefers HDR10 LEFT input color space for HDR10 passthrough
+    const DXGI_COLOR_SPACE_TYPE outputCS = DX::Windowing()->UseLimitedColor()
+                                               ? DXGI_COLOR_SPACE_RGB_STUDIO_G2084_NONE_P2020
+                                               : DXGI_COLOR_SPACE_RGB_FULL_G2084_NONE_P2020;
+
+    hr = m_pEnumerator1->CheckVideoProcessorFormatConversion(
+        DXGI_FORMAT_P010, DXGI_COLOR_SPACE_YCBCR_STUDIO_G2084_LEFT_P2020, format, outputCS,
+        &supported);
+    const bool HDRleft = SUCCEEDED(hr) && static_cast<bool>(supported);
+
+    hr = m_pEnumerator1->CheckVideoProcessorFormatConversion(
+        DXGI_FORMAT_P010, DXGI_COLOR_SPACE_YCBCR_STUDIO_G2084_TOPLEFT_P2020, format, outputCS,
+        &supported);
+    const bool HDRtopLeft = SUCCEEDED(hr) && static_cast<bool>(supported);
+
     m_BT2020TopLeft = !left && topLeft;
     CLog::LogF(LOGDEBUG,
                "BT.2020 input color spaces supported: YCBCR_STUDIO_G22_LEFT_P2020: {}, "
                "YCBCR_STUDIO_G22_TOPLEFT_P2020: {}",
                left ? "yes" : "no", topLeft ? "yes" : "no");
+
+    m_HDR10Left = !HDRtopLeft && HDRleft;
+    CLog::LogF(LOGDEBUG,
+               "HDR10 input color spaces supported with {} range output: "
+               "YCBCR_STUDIO_G2084_LEFT_P2020: {}, "
+               "YCBCR_STUDIO_G2084_TOPLEFT_P2020: {}. {} will be used.",
+               DX::Windowing()->UseLimitedColor() ? "limited" : "full", HDRleft ? "yes" : "no",
+               HDRtopLeft ? "yes" : "no",
+               m_HDR10Left ? "Fallback chroma siting left"
+                           : "Preferred UHD Blu-Ray chroma siting top left");
```

Comment:
Respectively "Fallback left chroma si**tt**ing" and "Preferred UDH Blu-Ray top left chroma si**tt**ing" please. (spelling +order of words in english)

Reply:
I changed order of words but spelling is correct siting  (one t) see:
https://learn.microsoft.com/en-us/windows/win32/api/dxgicommon/ne-dxgicommon-dxgi_color_space_type#siting

> The following examples show different combination of chroma siting in a 4x4 image.
https://www.kernel.org/doc/html/next/userspace-api/media/v4l/yuv-formats.html

Reply:
yes my bad, I assumed it was derived from sit, not site